moon/controller_hauler_round2.py

```diff
@@ -7,7 +7,7 @@
 import math
 from datetime import timedelta
 from statistics import median
-from random import getrandbits
+from random import getrandbits,randrange
```

Comment:
as discussed over the mail, use directly `self.rand.<function>` to guarantee repeatability ... and then you do not need to import it too.

Reply:
ok, removing imports